app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>RecipeApp</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= javascript_importmap_tags %>
+  </head>
+
+  <body>
+
+    <ul>
+      <li><%= link_to "Home", root_path %></li>
+      <li><%= link_to "Food List", foods_path %></li>
+      <li><%= link_to "Recipe List", recipes_path %></li>
+      <li><%= link_to "Public Recipes List", public_recipes_path %></li>
+    </ul>
```

Comment:
- [ ] Great progress on building your navbar! 👏 However, I have a few suggestions to enhance the functionality of your app:

 - **Update Navbar After Login/Sign-Up:** 
   When users successfully log in or sign up and are logged into your app, consider updating the navbar dynamically to improve user experience. Instead of displaying the `login` and `register` links, which might cause confusion for logged-in users, you could replace them with a `logout` link. This way, users can easily find the option to log out whenever they need to. Below for more details:
   
   #### Instead of 
   
<img width="1115" alt="Screenshot 2024-02-07 at 23 17 45" src="https://github.com/over-geek/RecipeApp/assets/107192076/4e01b171-cb37-4f03-b472-64ab93f4df65">
  

    #### Do this 
     
<img width="1115" alt="Screenshot 2024-02-07 at 23 18 01" src="https://github.com/over-geek/RecipeApp/assets/107192076/a23e0aab-7701-40e3-a72a-14fb19352355">


- Add a logout button

Happy coding! ✅😊👨‍💻